ACE/ace/CDR_Base.h

```diff
@@ -207,6 +207,14 @@ class ACE_Export ACE_CDR
   typedef ACE_UINT32 ULong;
   typedef ACE_INT64 LongLong;
   typedef ACE_UINT64 ULongLong;
+  typedef ACE_INT8 Int8;
+  typedef ACE_UINT8 Uint8;
```

Comment:
`Uint` or `UInt`, looking at the other typedefs I think we should go for `UInt`

Reply:
I didn't really notice the difference. I would say `Uint` if this was by itself, but yeah it should probably be `UInt` to match the others.